backend-elasticsearch/src/main/java/org/hibernate/search/v6poc/backend/elasticsearch/index/impl/IndexingBackendContext.java

```diff
@@ -47,7 +46,11 @@ public IndexingBackendContext(BackendImplementor<?> backend,
 
 	@Override
 	public String toString() {
-		return getClass().getSimpleName() + "[backend=" + backend + "]";
+		return getClass().getSimpleName() + "[" + failureContext + "]";
```

Comment:
Not a big fond of it. The `failureContext` could be different in the future from what we want in the `toString()`. I don't think having a strong link between both is such a good idea.

Reply:
Are you sure it's really important? I would expect the "failure context" used in the "Indexing backend context" to translate more or less accurately what we want in the `toString()` method... I mean, if it's good enough for users, it should be good enough for us.
The alternative is to keep a reference to a `BackendImplementor` object whose methods we don't use, which is really weird.